pipelines/preparation/preparation_matched_multi.yaml

```diff
@@ -3,5 +3,7 @@ tasks:
   matchCatalogsPatchMultiBand:
     class: lsst.faro.preparation.PatchMatchedMultiBandPreparationTask
     config:
-      connections.photoCalibName: fgcm_photoCalib
-      apply_external_wcs: True  # We only support jointcal for now
+      python: |
```

Comment:
Looking over the pipeline used by validate_drp_gen3, I don't think this will break that, but just want to make sure.

Reply:
Oh, good point. I'll look up what `validate_drp_gen3` uses.